pkg/cmd/server/start/start_master.go

```diff
@@ -614,10 +615,15 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		if err != nil {
 			glog.Fatalf("Could not get client for deployment controller: %v", err)
 		}
+
+		// TODO there has to be a better way to do this
+		oc.PrivilegedLoopbackClientConfig.ContentConfig.GroupVersion = &unversioned.GroupVersion{Group: "batch", Version: "v2alpha1"}
```

Comment:
don't mutate shared config in here. also, won't forcing to v2alpha1 break regular jobs?